modelimplementation/src/main/java/org/verapdf/model/tools/XMPChecker.java

```diff
@@ -0,0 +1,226 @@
+package org.verapdf.model.tools;
+
+import org.apache.log4j.Logger;
+import org.apache.pdfbox.cos.*;
+import org.apache.pdfbox.util.DateConverter;
+import org.apache.xmpbox.XMPMetadata;
+import org.apache.xmpbox.schema.AdobePDFSchema;
+import org.apache.xmpbox.schema.DublinCoreSchema;
+import org.apache.xmpbox.schema.XMPBasicSchema;
+import org.apache.xmpbox.xml.DomXmpParser;
+import org.apache.xmpbox.xml.XmpParsingException;
+
+import java.io.IOException;
+import java.util.*;
+
+/**
+ * Class for checking match of document information dictionary and xmp. Current class
+ * compare 8 predefined fields in info with document xmp:
```

Comment:
This class matches document information dictionary and xmp metadata by comparing eight predefined fields.